source/configuration.txt

```diff
@@ -114,6 +114,10 @@ by the {+product-short+}:
          its nested namespaces only
        - ``"None"``: Do not analyze LINQ expressions
        
+       .. important::
+
+          The ``LinqAnalysisVerbosity`` configuration option is only available for {+product-short+} v2.0 and above.
```

Comment:
S: style guide - earlier/later - https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#e:~:text=E-,earlier,-later%20Use%20these + placement of `only`
```suggestion
          The ``LinqAnalysisVerbosity`` configuration option is available only in {+product-short+} v2.0 and later.
```